### PR TITLE
add draft for new blog post about HypriotOS Barbossa

### PR DESCRIPTION
**Pull request by Govinda-Fichtner**
connects to hypriot/build-pipeline#190

### CODE REVIEW

File: content/post/hypriotos-barbossa-for-raspberry-pi-3.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+![Raspberry Pi 3](/images/hypriotos-barbossa/iceberg.jpg)
+<div style="text-align:right; font-size: smaller">Image courtesy of [David](https://www.flickr.com/photos/davidw/)</div>
+
+<!--more-->
+
+## Features at a glance
+
+* support for Raspberry Pi Zero, 1, 2 and 3
+* Linux kernel 4.1.19
+* Docker Engine 1.11.0
```
Comment (StefanScherer) [inline content/post/hypriotos-barbossa-for-raspberry-pi-3.md]: 1.11.1 ?

File: content/post/hypriotos-barbossa-for-raspberry-pi-3.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+* Docker Compose 1.7.1
+* Swarm 1.2.2
+* Cluster-Lab 0.2.6
+* device-init 0.1.5
+
+
+## Updated Docker and Docker-Tools
+Compared to our latest HypriotOS Berry 0.7 we did upgrade the Docker-Engine to v1.11 which now has several binaries instead of one.
+Besides the original Docker binary there is now also a Containerd and a Runc binary. Containerd is basically a supervisor for individual containers that are run by Runc.
```
Comment (StefanScherer) [inline content/post/hypriotos-barbossa-for-raspberry-pi-3.md]: s/Runc/runC/g

File: content/post/hypriotos-barbossa-for-raspberry-pi-3.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+With the help of our [flash-tool](https://github.com/hypriot/flash) it is really easy to bring a Raspberry Pi 3 online with only WiFi network connectivity:
+
+```
+flash --ssid MyNetworkName --password SomeSecret https://github.com/hypriot/image-builder-rpi/releases/download/v0.8.0/sd-card-rpi-v0.8.0.img.zip
+```
+
+After that you can usually connect to your Pi by:
+
+```
+ssh pirate@black-pearl
```
Comment (StefanScherer) [inline content/post/hypriotos-barbossa-for-raspberry-pi-3.md]: black-pearl.local ?

File: content/post/hypriotos-barbossa-for-raspberry-pi-3.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+```
+
+The password is 'hypriot'. And that's it.
+
+By the way this also works if you attach an external WiFi adapter to a Raspberry Pi Zero, 1 and 2.
+
+## Customizing your Raspberry Pi Configuration
+The ability to customize the hostname of your Pi, to add WiFi credentials, etc. is made possible with the help of device-init.
```
Comment (StefanScherer) [inline content/post/hypriotos-barbossa-for-raspberry-pi-3.md]: We should link to device-init GitHub repo and readme where user's can find more details about the device-init.yaml file.

File: content/post/hypriotos-barbossa-for-raspberry-pi-3.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+We are going to add more features - like adding public ssh-keys, setting the locale or timezone - to device-init step by step in the next weeks.
+
+## Hypriot Cluster-Lab
+We added the Hypriot Cluster-Lab to the image, too.
+
+It is disabled by default, but can easily be enabled when you flash the image:
+
+```
+flash --cluster-lab true https://github.com/hypriot/image-builder-rpi/releases/download/v0.5.6/sd-card-rpi-v0.5.6.img.zip
```
Comment (StefanScherer) [inline content/post/hypriotos-barbossa-for-raspberry-pi-3.md]: @Govinda-Fichtner Now as I see an example of the `--cluster-lab` option I think we could remove the `true` as there is only one option that flash can set. Without that option the cluster lab is disabled by default. Just the option that turns the feature on. WDYT?

Comment (Govinda-Fichtner) [inline content/post/hypriotos-barbossa-for-raspberry-pi-3.md]: @StefanScherer I think that OK. Go ahead please.

File: content/post/hypriotos-barbossa-for-raspberry-pi-3.md (hunk 4 above)
Comment (Govinda-Fichtner) [inline content/post/hypriotos-barbossa-for-raspberry-pi-3.md]: Yeah. Gonna add a link.

File: content/post/hypriotos-barbossa-for-raspberry-pi-3.md (hunk 3 above)
Comment (Govinda-Fichtner) [inline content/post/hypriotos-barbossa-for-raspberry-pi-3.md]: @StefanScherer That's not necessary for me. I always work without the .local part? Is it necessary for you?

Comment (StefanScherer) [inline content/post/hypriotos-barbossa-for-raspberry-pi-3.md]: Yes.

```
$ ping pi4
ping: cannot resolve pi4: Unknown host
~
$ ping pi4.local
PING pi4.local (x.x.x.x): 56 data bytes
64 bytes from x.x.x.x: icmp_seq=0 ttl=64 time=1.076 ms
```

**Comment by Govinda-Fichtner** [comment]
@StefanScherer Thanks for the feedback.

**Pull request merged**